### DISCUSSION TITLE
Example with Bioconda in the documentation

### DISCUSSION THREAD

**Discussion by apcamargo** [prefix-dev/pixi#2393] [Ideas]
After conda-forge, Bioconda is the largest Conda channel and the bioinformatics community using the Conda ecosystem is huge. I noticed that the biggest difficulty in trying to convince people to move to Pixi is that there's no example in the documentation showing how to add Bioconda to the list of default channels (either in a project or locally).

I think that it could be valuable to have a simple example in the documentation. On top of being useful to the Bioconda community, this could serve as a example of how to add channels (or any other configuration) to the global/project configuration.

**Comment by martin-g** [comment]
The solution is to add `bioconda` to the `channels` key in `pixi.toml` file:
```
channels = ["conda-forge", "bioconda"]
```

**Comment by ruben-arts** [comment]
Thanks for the tip, sorry for missing this discussion, if you search `bioconda` in the docs you should find multiple mentions now. There is also the command:
```
# Set it once
pixi config set default-channels '["conda-forge", "bioconda"]' --global
# Or extend existing list
pixi config append default-channels bioconda --global
```

Which will make sure that bioconda is added to all new projects run by `pixi init`

On `pixi init` you can also add them on demand e.g.:
```
pixi init --channel conda-forge --channel bioconda
```

In a project, you can run:
```
pixi project channel add bioconda 
```
Which will create the result @martin-g mentioned.

**Comment by markusschlenker** [comment]
Is there a conda-pypi-map file for the bioconda channel, or should I just use the same as for conda-forge?
